py/fastspecfit/continuum.py

```diff
@@ -180,9 +180,11 @@ def __init__(self, metallicity='Z0.0190', minwave=None, maxwave=6e4, seed=1):
                                                  'wise2010-W1', 'wise2010-W2')
 
         # rest-frame filters
-        self.absmag_bands = ['u', 'g', 'r', 'i', 'z', 'W1']
+        self.absmag_bands = ['u', 'g', 'r', 'i', 'z', 'W1', 'UX', 'B', 'V']
         self.absmag_filters = filters.load_filters('decam2014-u', 'decam2014-g', 'decam2014-r',
-                                                   'decam2014-i', 'decam2014-z', 'wise2010-W1')
+                                                   'decam2014-i', 'decam2014-z', 'wise2010-W1',
+                                                   'bessell-U', 'bessell-B', 'bessell-V',
+                                                    )
 
```

Comment:
Thanks @biprateep but I'm not crazy about the `UX` notation. What if we changed the `ugriz` photometry from DECam to SDSS and then did something like:
```
self.absmag_bands = ['sdss_u', 'sdss_g', 'sdss_r', 'sdss_i', 'sdss_z', 'W1', 'U', 'B', 'V']
```
?

I also wonder if we could specify the band shift here and propagate it into the `kcorr_and_absmag` method: https://github.com/desihub/fastspecfit/blob/main/py/fastspecfit/continuum.py#L856.

If we can pull this off, we should use `z=0.1` for the SDSS bands (so we can easily compare luminosity functions to SDSS) and `z=0.0` for `W1` and `UBV`.

Reply:
Hi @moustakas , I do not have strong opinions about the naming, so I agree with your suggestions. I will go ahead and change the band filters and incorporate the band shifts as you suggested.

But I believe rather than hard coding band shifts, etc it is better to give people the options to choose. It might be useful to handle this situation using a config file which contains the list of filters and band shifts. But this is the topic of another pull request. For now, I will go ahead with your suggestion of sdss bands + band shifts